Tests/OpenAPIRuntimeTests/Conversion/Test_Converter+Client.swift

```diff
@@ -120,6 +120,30 @@ final class Test_ClientConverterExtensions: Test_Runtime {
         try await XCTAssertEqualStringifiedData(body, testStructPrettyString)
         XCTAssertEqual(headerFields, [.contentType: "application/json", .contentLength: "23"])
     }
+    
+    //    | client | set | request body | XML | optional | setOptionalRequestBodyAsXML |
```

Comment:
Reminder: Please adapt the `Converting-between-data-and-Swift-types.md` docs page with the addition of XML in your swift-openapi-generator PR, when it comes to it.